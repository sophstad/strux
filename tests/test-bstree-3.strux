int main() {
    BSTree::int tree = new BSTree::int();
    tree.add(1);
    tree.add(2);
    tree.add(3);
    tree.add(0);
    tree.add(-4);
    tree.add(-2);
    tree.add(-3);
    tree.show();
    return 0;
}