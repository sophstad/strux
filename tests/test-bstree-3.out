                    (001)                                                       
                 +----+----+                                                    
               (000)     (002)                                                  
  +--------------+         +----+                                               
(-04)                         (003)                                             
  +---------+                                                                   
          (-02)                                                                 
       +----+                                                                   
     (-03)                                                                      
                                                                                
                                                                                
                                                                                
                                                                                
                                                                                
                                                                                
                                                                                
                                                                                
                                                                                
                                                                                
                                                                                
